src/styles/blocks/_nav.scss

```diff
@@ -0,0 +1,54 @@
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__list {
+    @include main-text;
+    display: flex;
+    height: 100%;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $main-color-accent;
+
+    cursor: pointer;
+
+    transition: color 0.3s;
+
+    &:hover {
+      color: $blue-color-accent;
+    }
+  }
+}
+
+.is-active {
```

Comment:
will be better to create active modifier for the links, and move these styles to those modifier
and use is-active class only in html for tests passing